qemu/tests/cfg/vm_poisoned_migration_prevention.cfg

```diff
@@ -0,0 +1,16 @@
+- vm_poisoned_migration_prevention:
+    only Linux
+    required_qemu = [9.0.0,)
+    type = vm_poisoned_migration_prevention
+    virt_test_type = qemu
+    smp = 8
+    vcpu_maxcpus = ${smp}
+    slots_mem = 4
+    maxmem_mem = 32G
+    mem = 4096
+    kill_vm_on_error = yes
+    mig_timeout = 1200
+    migration_protocol = "tcp"
+    # The user needs to provide here a valid path to a file that creates HWPoison in the host
+    hwpoison_file = ""
+    mnt_dir = "/var/tmp/hwpoison"
```

Comment:
Nothing related with mount,  suggest `target_path`

Reply:
Updated !

Reply:
Seems you did not update the variable name.

Reply:
Sorry for that, name updated